### DISCUSSION TITLE
在不同链上传相同的文件，会出现什么情况？

### DISCUSSION THREAD

**Discussion by linghuccc** [IntensiveCoLearning/Web3-URL#211] [Q&A] [answered]
我先是在 sepolia 链上 “ethfs-cli create” 了一个 FlatDirectory Address （Address1），用他 “ethfs-cli upload” 了一部分文件之后，发现 eth 不够了，就停了下来，然后在 W3Q Galileo 链上 “ethfs-cli create” 了另一个 FlatDirectory Address （Address2），用他重新 “ethfs-cli upload” 先前的文件，发现已经上传成功的文件，不需要再传了。

那这样的话，先前上传的文件，是不是只能通过 https://Address1.3333.w3link.io/ 才能查到？我现在想在 Address2 查看用Address1 上传的文件，怎么办？

**Comment by qzhodl** [comment, marked as answer]
有些困惑，如果是重新在W3Q Galileo 链上创建了一个FlatDirectory，并且向这个地址upload的话，应该需要重新上传所有的文件的。可能需要检查一下，看看是不是什么参数传递的有些问题。

**Reply by linghuccc** [reply]
> 有些困惑，如果是重新在W3Q Galileo 链上创建了一个FlatDirectory，并且向这个地址upload的话，应该需要重新上传所有的文件的。可能需要检查一下，看看是不是什么参数传递的有些问题。

那如果我在 W3Q Galileo 链上再建一个FlatDirectory（Address 3），然后用这个地址上传的话，是不是还需要重新上传一遍所有文件？

**Reply by qzhodl** [reply]
是的

**Reply by linghuccc** [reply]
> 是的

明白了。而且不管我在哪条链上传（FlatDirectory 在哪条链），文件都是传到 3333 链上，所以最终查看文件，都是https://FlatDirectoryAddress.3333.w3link.io/文件名，是吧？

**Reply by qzhodl** [reply]
W3Q Galileo 查看的chain id是 3334，Sepolia 查看的chain id 是 3333

**Reply by linghuccc** [reply]
> W3Q Galileo 查看的chain id是 3334，Sepolia 查看的chain id 是 3333

明白了，谢谢
